The declaration of asprintf seems to be causing compilation issues on musl
systems. I believe this is because char *fmt isn't being declared as const in
the function declaration, and it should be.
(See asprintf man page: https://man7.org/linux/man-pages/man3/asprintf.3.html)
(PR submitted upstream: https://github.com/kmatheussen/jack_capture/pull/45)
--- jack_capture-e2c0c3d25fce0a13f3f18c622079cd29178af7f8/jack_capture.c
+++ jack_capture-e2c0c3d25fce0a13f3f18c622079cd29178af7f8/jack_capture.c
@@ -919,7 +919,7 @@
 #define vsnprintf _vsnprintf
 #endif
 #ifndef __APPLE__
-int asprintf(char **buffer, char *fmt, ...) {
+int asprintf(char **buffer, const char *fmt, ...) {
     /* Guess we need no more than 200 chars of space. */
     int size = 200;
     int nchars;
